libs/dh/metering-point/feature-overview/src/components/dh-metering-point-actions.component.ts

```diff
@@ -127,10 +134,25 @@ export class DhMeteringPointActionsComponent {
   connectionState = input<ConnectionState | null>();
   installationAddress = input<InstallationAddress | null>();
 
+  protected readonly hasAllowedMarketRolesForSendMeasurementsRequest = toSignal(
+    combineLatest([
+      this.permissionService.hasMarketRole(EicFunction.MeteredDataResponsible),
```

Comment:
nitpick: `measurements:manage` permission cannot be assigned to roles under `MeteredDataResponsible` and `Delegated` so the permission itself will not allow these market roles to see the button. So these are not necessary.